packages/mui-base/src/InputUnstyled/useInput.test.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { createRenderer } from 'test/utils';
+import { expect } from 'chai';
+import useInput from './useInput';
+import { useSlotProps } from '../utils';
+
+describe('useInput', () => {
+  const { render } = createRenderer();
+  describe('params: inputRef', () => {
+    it('should be able to attach input ref passed through params', () => {
```

Comment:
This test could be added to #35807. Could you please work with @sai6855 to include it?
The next test case should be covered by testing the InputUnstyled, so it's not necessary.